bin/pull_env

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
```

Comment:
Can refactor these bash scripts into ruby later if needed, for now I know they work.

Reply:
Bash good with me 👍